pkg/operator/ceph/object/spec.go

```diff
@@ -468,15 +468,11 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) (v1.Container,
 	// set RGW read Affinity
 	isCrushLocationSet := false
 	if c.store.Spec.Gateway.ReadAffinity != nil {
-		if c.clusterInfo.CephVersion.IsAtLeastTentacle() {
```

Comment:
How about if we instead change this to `IsAtLeastSquid()`? That way it's just less risk of merge conflicts with resyncs

Reply:
done.